CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 New Features
 ------------
-- twilio-video.js now supports WKWebView and SFSafariViewController on the latest version of iOS. Thank you @codeborne for your contributions. (VIDEO-8374)
+- twilio-video.js now supports WKWebView and SFSafariViewController on iOS versions 15.2 and up. Applications will need to include the camera usage description, microphone usage description, inline media playback on WKWebviews, and a [User-Agent string](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) that follows the correct format. Thank you @codeborne for your contributions. (VIDEO-8374)
```

Comment:
It might make sense to explicitly state that the first 3 items are iOS app requirements. And the last item (user agent string) is a web app requirement. But your call. Otherwise it is accurate.